src/GameLogic/PlayerActions/ItemConsumeActions/ItemUpgradeConsumeHandlerPlugIn.cs

```diff
@@ -166,7 +166,7 @@ private bool TryAddItemOption(Item item, IContext persistenceContext)
             else
             {
                 // ItemOptionTypes.Option
-                optionLink.ItemOption = possibleOptions.First();
+                optionLink.ItemOption = possibleOptions.SelectRandom();
```

Comment:
I fucked this in the previous PR :smiley:. This was not randomizing options for the wings that have several possible options.